framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -484,10 +485,16 @@ public Boolean visitDeclared_Declared(
   /**
    * A helper class for visitDeclared_Declared. There are subtypes of DefaultTypeHierarchy that need
    * to customize the handling of type arguments. This method provides a convenient extension point.
+   *
+   * @param subtype a possible subtype
+   * @param supertype a possible supertype
+   * @param subtypeRaw whether {@code subtype} is a raw type
+   * @param supertypeRaw whether {@code supertype} is a raw type
+   * @return the result of visiting type args
    */
   protected boolean visitTypeArgs(
-      final AnnotatedDeclaredType subtype,
-      final AnnotatedDeclaredType supertype,
+      AnnotatedDeclaredType subtype,
+      AnnotatedDeclaredType supertype,
```

Comment:
Just below there is 
```java
 if (ignoreTypeArgs) {
      return true;
    }
```
Why is no test needed for the raw component?  What if this method were called with two arguments:  raw `Class` and raw `List`?  I suspect that `subtype` and `supertype` must have some specific relationship, such as that their raw types are the same or in some subtyping relationship.  Could you document that?

Reply:
I updated the method's Javadoc.  The underlying type of subtype is a subtype of the underlying type of supertype.  Also, this method is only checking the type arguments.